src/test/java/sg/dex/starfish/samples/AssetSample.java

```diff
@@ -43,7 +43,7 @@ private static void createAssetFromPath() throws IOException {
 
         String path = "example/software_training_data.pdf";
         //System.out.println("local file :" + path);
-        Asset assetPath = ResourceAsset.create(path);
+        Asset assetPath = ResourceAsset.create(path,false);
```

Comment:
The content hash argument should be optional (default false), so you shouldn't need to include it in most cases.